kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -1,6 +1,8 @@
-ChannelRouter = Backbone.Router.extend({
+TopicChannelRouter = Backbone.Router.extend({
     initialize: function(options) {
+        _.bindAll(this);
         this.default_channel = options.default_channel;
+        $("#nav_learn").click(this.intercept_learn_nav);
```

Comment:
Doesn't work. Will leave this as it is until we have a more general framework for routing across the app.